src/ansys/fluent/core/_stand_alone_datamodel_client/_preferences_client.py

```diff
@@ -0,0 +1,93 @@
+"""Sample testing code (next)."""
```

Comment:
@prmukherj just wondering if we need all this code

Reply:
@seanpearsonuk, yes something like that can be done. I just checked it and it seems to be working. I'll update this code as required. Thank you.

Reply:
@seanpearsonuk and @mkundu1, does the current implementation look good?

Reply:
@prmukherj please go ahead and merge. Thanks.

Reply:
Next set of changes after merge, based on this kind of thing:
```
>>> from ansys.fluent.core.services.datamodel_se import DatamodelService, PyMenuGeneric
>>> pref_root = PyMenuGeneric(s.fluent_connection.datamodel_service_se, "preferences")
>>> pref_root.MeshingWorkflow()
{'CheckpointingOption': 'Write mesh files', 'DrawSettings': {'FaceZoneLimit': 10000, 'AutoDraw': 'auto', 'FacetLimit': 5000000}, 'SaveCheckpointFiles': True, 'TempFolder': '', 'TemplatesFolder': '', 'Verbosity': 'off'}
>>>
```